fhir-persistence-cassandra/pom.xml

```diff
@@ -0,0 +1,116 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>fhir-persistence-cassandra</artifactId>
```

Comment:
we should add this new module to the top-level README tables, probably under the `Server` section:  https://github.com/IBM/FHIR/blob/main/README.md#server

Reply:
What does this module and the -cos module actually do?